cli/README.md

```diff
@@ -1,55 +1,74 @@
-# Xata CLI
+# Command Line Interface (CLI)
 
-## Installing
+Xata provides a CLI as a convenience for developers. It is intended to provide feature parity with Xata's [Web UI](https://docs.xata.io/getting-started) (User Interface) to enable developers to build applications at **rapid pace** with **as little friction as possible**.
 
-The recommended way of installing the CLI is by installing it as an npm package globally:
+## Installation
 
-```bash
-npm install @xata.io/cli -g
+Our CLI is distributed both as an **[npm](https://www.npmjs.com/package/@xata.io/cli) package** for developers who work with Node.js, and a **cross-platform binary** for those who don't. Below, you'll find commands that you can copy and paste into your platform's shell that will help you install the CLI on your specific platform.
```

Comment:
> developers who work with Node.js

Not sure about that phrasing, because one might be a web developer, not using Node.js at all directly, and have npm installed.

Reply:
Thanks for this. Do you have a suggestion? How about this:

```suggestion
Our CLI is distributed both as an **[npm](https://www.npmjs.com/package/@xata.io/cli) package** for JavaScript developers, and a **cross-platform binary** for those who don't. Below, you'll find commands that you can copy and paste into your platform's shell that will help you install the CLI on your specific platform.
```

Reply:
Or even this:


```suggestion
Our CLI is distributed both as an **[npm](https://www.npmjs.com/package/@xata.io/cli) package** for developers who have Node.js installed on their systems, and a **cross-platform binary** for those who don't. Below, you'll find commands that you can copy and paste into your platform's shell that will help you install the CLI on your specific platform.
```

Reply:
Hmmm, based on https://github.com/xataio/client-ts/pull/400#pullrequestreview-1032729227, this maybe a bit short-sighted. How about:

```suggestion
Our CLI is distributed as an **[npm](https://www.npmjs.com/package/@xata.io/cli) package**. Here's how you can install it.
```

Reply:
Done in https://github.com/xataio/client-ts/pull/400/commits/137212cf19d8d42871a64ffac6acaa398d52ef7c